kotlin/change-viewpoint/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <androidx.appcompat.widget.AppCompatButton
```

Comment:
I think these buttons are perfectly functional for the UI, mostly because they keep the code super clean. The only sleek material design idea I could come up with would be a speed dial: https://medium.com/@lcdsmao/material-design-transition-for-floating-action-button-in-android-dc8304343cfe

Up to you if that's worth it. Otherwise, I would still probably make these buttons a tiny bit smaller and move them to the bottom right if possible, just because it reduces the amount of thumb-stretching you need to do to hit them.